browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -545,6 +547,16 @@ void RewardsDOMHandler::OnGetRewardsParameters(
         "brave_rewards.rewardsParameters", data);
 }
 
+void RewardsDOMHandler::OnRewardsInitialized(
+    brave_rewards::RewardsService* rewards_service) {
+  if (!web_ui()->CanCallJavascript())
+    return;
+
+  web_ui()->CallJavascriptFunctionUnsafe(
+      "brave_rewards.initialized",
+      base::Value(0));
```

Comment:
I know the existing call to "initialized" provides a zero (I guess to mimic a `LEDGER_OK` value), so, although the value isn't used, I think it's fine if we want to be consistent.